sabnzbd/api.py

```diff
@@ -629,10 +629,11 @@ def _api_warnings(name, kwargs):
 LOG_USER_RE = re.compile(rb"(user|username)\s?=\s?[\S]+", re.I)
 LOG_PASS_RE = re.compile(rb"(password)\s?=\s?[\S]+", re.I)
 LOG_INI_HIDE_RE = re.compile(
-    rb"(email_pwd|email_account|email_to|pushover_token|pushover_userkey|pushbullet_apikey|prowl_apikey|growl_password|growl_server|IPv[4|6] address)\s?=\s?[\S]+",
+    rb"(email_pwd|email_account|email_to|email_from|pushover_token|pushover_userkey|pushbullet_apikey|prowl_apikey|growl_password|growl_server|IPv[4|6] address)\s?=\s?[\S]+",
     re.I,
 )
 LOG_HASH_RE = re.compile(rb"([a-fA-F\d]{25})", re.I)
+LOG_FEED_API_RE = re.compile(rb"(uri\s=\s.*r=)([a-z0-9]{32})", re.I)
```

Comment:
Why would these not be caught by the line above it?

Reply:
Different hash format. The current one does match most things but failed on this one because geek doesnt do hex so you need a-z to cover it. Since I didn't want to change the existing hash one to be even more zealous on matching I figure it was low stakes to just target rss uri as a last resort catch

Reply:
Can you give an example of the API key?

Reply:
`mRu0nMyGsm5wyBFrprcU7L71GUBF2Qcg`

Reply:
Ah, now I see. Then let's just update the existing one to include `a-z` so it's universal.

Reply:
will do

Reply:
changing existing hash format to a-z makes it a bit over zealous for my taste...   thoughts?
```
2023-03-09 10:52:42,050::DEBUG::[deobfuscate_filenames:111] Checking: <HASH>WENLE
2023-03-09 10:52:42,050::DEBUG::[deobfuscate_filenames:167] Obfuscated (default)
2023-03-09 10:52:42,054::DEBUG::[deobfuscate_filenames:111] Checking: <HASH>ggm3d
2023-03-09 10:52:42,055::DEBUG::[deobfuscate_filenames:167] Obfuscated (default)
2023-03-09 10:52:42,057::DEBUG::[deobfuscate_filenames:111] Checking: <HASH>donjx
2023-03-09 10:52:42,057::DEBUG::[deobfuscate_filenames:167] Obfuscated (default)
2023-03-09 10:52:42,060::DEBUG::[deobfuscate_filenames:111] Checking: <HASH>GQ
2023-03-09 10:52:42,060::DEBUG::[deobfuscate_filenames:167] Obfuscated (default)
2023-03-09 10:52:42,062::DEBUG::[deobfuscate_filenames:111] Checking: <HASH>6aa163e
...
2023-03-09 19:19:37,392::INFO::[nzbstuff:1740] Detected filename based on par2: We.Baby.Bears.S01E37.Lighthouse.1080p.CN.WEB-DL.AAC2.0.x264-TCV.part07.rar -> <HASH>5U0Cn.part07.rar
2023-03-09 19:19:37,393::DEBUG::[assembler:79] Decoding part of \\?\C:\Users\<USERNAME>\downloads\incomplete\We.Baby.Bears.S01E37.1080p.CN.WEB-DL.AAC2.0.x264-TCV\<HASH>5U0Cn.part07.rar
...
2023-03-12 01:30:39,760::DEBUG::[newsunpack:2006] Quick-check of file <HASH>8.part22.rar OK
2023-03-12 01:30:39,760::DEBUG::[newsunpack:2006] Quick-check of file <HASH>8.part13.rar OK
2023-03-12 01:30:39,761::DEBUG::[newsunpack:2006] Quick-check of file <HASH>8.part15.rar OK
```

Reply:
That’s fine, they are hashes anyway.